pallets/ethereum-events/src/lib.rs

```diff
@@ -1672,39 +1652,53 @@ enum Releases {
     Unknown,
     V2_0_0,
     V3_0_0,
+    V4_0_0,
 }
 
+//Todo: Change this once merged
 impl Default for Releases {
     fn default() -> Self {
-        Releases::V3_0_0
+        Releases::V4_0_0
     }
 }
 
 pub mod migrations {
     use super::*;
-    use frame_support::{migration::storage_key_iter, Blake2_128Concat};
-    pub type MarketplaceId = u32;
+    use frame_support::pallet_prelude::Weight;
+
+    pub fn get_migration_address<T: Config>() -> H160 {
+        let val_manager = ValidatorManagerContractAddress::<T>::get();
+        let lifting = LiftingContractAddress::<T>::get();
+        let pub_root = PublishRootContract::<T>::get();
```

Comment:
i would split this migration in 2 and only deal with val_mgr and lifting here. We can do a similar migration from eth transactions.

Reply:
Unless we remove the pallet dependencies again after the migration has run.

Reply:
Thank you for the review, will do this